data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -18,7 +20,11 @@ MACHINE_CONFIG_KUBE_CLIENT_AGENT_IMAGE=$(image_for kube-client-agent)
 MACHINE_CONFIG_INFRA_IMAGE=$(image_for pod)
 
 KUBE_ETCD_SIGNER_SERVER_IMAGE=$(image_for kube-etcd-signer-server)
-
+if image_for cluster-etcd-operator; then
```

Comment:
@hexfusion for openshift/cluster-etcd-operator#29 to work, wondering if we need to change this condition to something like the following psuedocode:

1) render ceo etcd CR with this, https://github.com/openshift/cluster-etcd-operator/pull/29/files#diff-02b2f9845fcac72e098f52904793e3ebR8
2) if `Managed`, set the `CLUSTER_ETCD_OPERATOR_IMAGE=$(image_for cluster-etcd-operator)` else `CLUSTER_ETCD_OPERATOR_IMAGE=""`

Reply:
will follow up a PR for this, marking it as resolved right now